spec/controllers/v2/rules_controller_spec.rb

```diff
@@ -78,6 +78,10 @@
           profile_id: parent.id
         ).sort_by(&:id)
       end
+      let(:invalid_params) do
+        parent = FactoryBot.create(:v2_profile)
+        { security_guide_id: parent.security_guide.id, profile_id: parent.id }
```

Comment:
Hm, I am wondering if the `security_guide_id` should be from the same hierarchy as the `profile`. I think it has a better coverage if just one of them is inconsistent.

Reply:
Just for clarity: this test only exchanged one ID at a time. 
This will be changed in the next push.